plugins/funind/invfun.ml

```diff
@@ -67,7 +67,7 @@ let observe_tac s tac g =
 let nf_zeta =
   Reductionops.clos_norm_flags  (CClosure.RedFlags.mkflags [CClosure.RedFlags.fZETA])
     Environ.empty_env
-    Evd.empty
+    (Evd.from_env Environ.empty_env)
```

Comment:
I don't think this change does anything.
Well, I guess it makes it easier to find other Evd.empty.

Reply:
I know it doesn't change much, but there are two cleanups needed. One is to remove `Evd.empty` so that we get universe constraints right in general. The other is to stop passing around empty maps, to get the evar substitution right. This PR is a step only for the first cleanup.

Reply:
@maximedenes : maybe I'm wrong but I don't think the evd change anything here since simple zeta normalize the term and, as far as I understand things, the evars and universes constraints are already solved in the term before its reduction. 
If this vision is false then the modif in get_body (file plugins/funind/functional_principles_proofs.ml) can change the behavior of function, an I will need more tests before approving the patch here.